src/ResponseTagger.php

```diff
@@ -36,10 +45,21 @@ class ResponseTagger
      * Constructor
      *
      * @param TagsInterface $client
+     * @param array         $options supported options:
+     *                               - strict (bool) Default: false. If set to true, throws exception when adding empty tags.
      */
-    public function __construct(TagsInterface $client)
+    public function __construct(TagsInterface $client, array $options = array())
     {
         $this->client = $client;
+
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults(array(
+            'strict' => false,
+        ));
+
+        $resolver->setAllowedTypes('strict', 'bool');
```

Comment:
ups. but this is a new major version, so lets just require at least 2.7 of options resolver in composer.json. please also adjust .travis.yml where it says symfony 2.3 atm.